cfgmgr/buffermgr.cpp

```diff
@@ -34,6 +34,8 @@ void BufferMgr::readPgProfileLookupFile(string file)
     ifstream infile(file);
     if (!infile.is_open())
     {
+        m_pgfile_processed = false;
```

Comment:
Thanks for making the changes.

You can move the above line to the beginning of the function.